### ISSUE TITLE
[Suggestion] Compatability with halfdoors?

### ISSUE THREAD

**Issue by netcore10** [InsomniaKitten/Couplings#22]
This would be great~

**Comment by ChloeDawn** [comment]
What is half doors? Please provide more information for your suggestion.

**Comment by netcore10** [comment]
Half doors is a mod! Couplings doesn't work with it, probably because it's not vanilla like doors https://www.curseforge.com/minecraft/mc-mods/half-doors (1.18.2)

**Comment by Kasprenius** [comment]
Along the same line, I would live compatibility for Dramatic Doors!
https://www.curseforge.com/minecraft/mc-mods/dramatic-doors

**Comment by LadyValishade** [comment]
agreed! getting this to work with those door mods would be super cool! Huge Vouch right here!

**Comment by Happyfrozenfire** [comment]
Currently, it seems to work with Dramatic Doors, but not Half Doors.
